server/Customer.java

```diff
@@ -4,15 +4,19 @@
 
 public class Customer {
 
-	private ArrayList<Account> accounts;
+	private ArrayList<newbank.server.Account> accounts;
+	String username;
+	String password;
 
-	public Customer() {
+	public Customer(String username, String password) {
 		accounts = new ArrayList<>();
+		this.username = username;
+		this.password = password;
 	}
 
 	public String accountsToString() {
 		String s = "";
-		for (Account a : accounts) {
+		for (newbank.server.Account a : accounts) {
```

Comment:
The `newbank.server` is not necessary, and actually makes the code harder to read, please remove it